include/armpp/hal/registers.hpp

```diff
@@ -22,10 +22,10 @@ concept register_value = integral<T> || enumeration<T>;
 
 namespace armpp::hal {
 
-constexpr std::size_t register_bits = 32;
+using raw_register                  = std::uint32_t;
```

Comment:
**Suggestion:** Consider using the 'using' directive for the register field aliases.
```suggestion
+using raw_register = std::uint32_t;
+using address      = std::uint32_t;
```